### ISSUE TITLE
Replace progress bars on about page with prose

### ISSUE THREAD

**Issue by yeldiRium** [yeldiRium/yeldirium.de#2]
The progress bars might have been a fancy idea at some point, but they are pretty inexpressive and they are very inaccessible.

Text is much more expressive, maintainable and accessible using e.g. a screenreader. Also the progress bars require custom HTML and CSS which does not work well with markdown content.

- [ ] Write prose to replace progress bars
- [ ] Remove progress bar HTML
- [ ] Remove corresponding CSS

**Comment by yeldiRium** [comment]
Done in ca3e9344a22b073f40e8799fe32908b1da41a20f.

**Comment by yeldiRium** [comment]
Obsolete CSS was removed in 333a11d3a94adaea9a14c5c748fbddf9468c9ab9.
